CHANGELOG.md

```diff
@@ -1,7 +1,288 @@
+  - **Potentially breaking change**: `ALPAKA_FN_INLINE` now enforces inlining for platforms other than CUDA and HIP #1918
+  - Replaced `alpaka::core::ConcurrentExecPool` with `alpaka::core::CallbackThread` in all queue implementations #1870
+  - If no back-end is enabled, alpaka automatically selects the serial back-end for examples and test cases #1843
+  - On Linux platforms, the free global memory is now determined by a call to `sysconf(_SC_AVPHYS_PAGES)` instead of querying `/proc/sysinfo` #1776
+  - **Potentially breaking change**: Changed CMake's look-up of MSVC's runtime libraries (see [here](https://cmake.org/cmake/help/v3.22/policy/CMP0091.html) for an in-depth explanation) #1751
+  - Unified `alpaka::{memcpy,memset}`'s internal `static_assert`s #1748
+  - `alpaka::core::aligned{Alloc,Free}` now internally use aligned `new`/`delete` instead of OS-specific APIs #1689
+- CUDA/HIP back-end changes:
+  - `nvcc` now makes correct use of `--Werror` and more CUDA-related warnings #2135
+  - Unified `ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK` macros #2090
+  - Made some internal constants `constexpr` #2063
+  - The CUDA/HIP back-ends will now always use `std::size_t` for internal pitch calculations #2056
+  - **Breaking change**: clang as CUDA compiler will only work in `Release` build mode #2027
+  - **Potentially breaking change**: In debug mode `ǹvcc` will now use the `-G` flag which enables device-side debug symbols #1977
+  - Starting from HIP 5.2.0, the HIP back-end includes `<hip/hiprand_kernel.h>` instead of `<hiprand_kernel.h>` #1914
+  - Starting from HIP 5.2.0, the HIP back-end makes use of `hip{Malloc,Free}Async` #1894
+  - If clang is used as CUDA compiler together with CUDA 11.3 a warning will be printed #1890
+  - Starting from HIP 5.4.0, the HIP back-end internally uses `hipLaunchHostFunc` instead of a work-around #1883
+  - Adapted to API change of CUDA 11.7 stream memory operations #1878 #1919 
+  - Shortened mangled CUDA kernel names #1795
+  - CUDA runtime versions checks are now based upon `CUDART_VERSION` instead of `BOOST_LANG_CUDA` #1777
+  - Because of a HIP performance regression the HIP back-end now uses the emulated `atomicAdd(float)` on the `Threads` hierarchy level #1771
+  - Changed look-up of built-in and emulated atomic functions for the CUDA and HIP back-ends #1768
+  - The HIP back-end now uses the built-in `atomicAdd(double)` #1767
+  - CUDA/HIP queues now internally make use of callback threads #1719 #1735 #1976 #2011
+- SYCL back-end changes:
+  - Removed unnecessary `-fintelfpga` flag from CMake build system when compiling the SYCL back-end for Intel FPGAs #2179
+  - **Breaking change**: Support for the `activemask` intrinsic is disabled for the SYCL back-end #2161
+  - Updated `README_SYCL.md` #2140
+  - **Breaking change**: Reworked CMake handling for SYCL targets #1970 #2066
+  - SYCL runtime objects are now `static` #1865
```

Comment:
This was later changed again when platforms became objects, so this no longer applies. Maybe delete it?